includes/admin/class-admin.php

```diff
@@ -40,7 +40,7 @@ protected function __construct( $plugin_base_file ) {
 		$this->plugin_basename = plugin_basename( $plugin_base_file );
 
 		// Load plugin's textdomain (i.e. translations)
-		load_plugin_textdomain( 'hm-content-toc', false, $this->plugin_basename . '/languages' );
+		load_plugin_textdomain( 'hm-content-toc', false, $this->plugin_basename . '/languages/' );
 
 		// Setup strings used extensively throughout the class
 		$this->page_title = __( 'HM Content TOC Settings', 'hm-content-toc' );
```

Comment:
Yep, that's fine. The [Pig Latin plugin](https://wordpress.org/plugins/piglatin/) is a good one for testing that all your strings are being translated.